gulpfile.js

```diff
@@ -158,8 +158,8 @@ function protractor() {
 }
 
 function watch() {
-	gulp.watch('src/**/*.ts', gulp.series(tslint, ts, 'unit'));
-	gulp.watch('src/**/*.less', gulp.series(lesslint, less));
+	gulp.watch('src/app/**/*.ts', gulp.series(tslint, ts, 'unit'));
+	gulp.watch('src/less/**/*.less', gulp.series(lesslint, less));
```

Comment:
Update `less` folder, task `lesslint` and `less`.
Also; `lesslint` needs to be rewritten using a `sasslint` plugin.